src/java/org/apache/nutch/protocol/RobotRulesParser.java

```diff
@@ -376,13 +379,18 @@ public int run(String[] args) {
    */
   private static class TestRobotRulesParser extends RobotRulesParser {
 
-    public TestRobotRulesParser(Configuration conf) {
+    public void setConf(Configuration conf) {
       // make sure that agent name is set so that setConf() does not complain,
```

Comment:
Hi @sebastian-nagel, a small question. Is this comment still valid, since if the user-agent is not provided we will check with robots.agent and others right?

Reply:
Thanks. You're right the comment wasn't up-to-date. Would have been simpler to drop the command-line overwrite of the checked agent names and rely only on properties. But I didn't want to break the existing behavior.